lib/mix/tasks/surface/surface.format.ex

```diff
@@ -2,8 +2,10 @@ defmodule Mix.Tasks.Surface.Format do
   @shortdoc "Formats Surface ~F sigils and .sface files in the given files/patterns"
 
   @moduledoc """
-  **To format Surface code using Elixir 1.13 or later, use
-  `Surface.Formatter.Plugin`.**
+  > #### Formatter Plugin Preferred {: .tip}
+  >
+  > To format Surface code using Elixir 1.13 or later, use
+  > `Surface.Formatter.Plugin`.
```

Comment:
After we update to the latest `ex_doc`, this admonition block will render like this:

![Screen Shot 2022-08-29 at 2 49 47 PM](https://user-images.githubusercontent.com/4662353/187305643-1ecb8c03-11bf-4e2b-9f3c-22750223906c.jpg)